express-notes-api/data.json

```diff
@@ -0,0 +1,24 @@
+{
+  "nextId": 20,
+  "notes": {
+    "12": {
+      "content": "The event loop, this, closures, and prototypal inheritance are special about JavaScrip.",
+      "Id": 12
+    },
+    "13": {
+      "content": "The event loop, this, closures, and prototypal inheritance are special about JavaScrip.",
+      "Id": 13
+    },
+    "16": {
+      "content": "Inertia is a property of matter.",
+      "Id": 16
+    },
+    "17": {
+      "content": "Inertia is a property of matter.",
+      "Id": 17
+    },
+    "content": {
+      "content": "The event loop, this, closures, and prototypal inheritance are special about JavaScrip."
+    }
```

Comment:
The content you write to `data.json` should have the same format as the pre-existing records.